pkgr.yml

```diff
@@ -11,7 +11,7 @@ Packages:
 
 Repos:
   - CRAN: https://cran.rstudio.com # used for most things
-  - MPN: https://mpn.metworx.com/snapshots/stable/2022-02-11 # used for pmtables and mrgval
+  - MPN: https://mpn.metworx.com/snapshots/stable/2023-08-24 # used for pmtables and mrgval
```

Comment:
Given the mpn.scorecard transition, this cleanup series could also 1) remove mrgval* from DESCRIPTION, 2) removed the mention here, 3) delete inst/validation, and 4) remove test IDs from the test anmes.

Reply:
> I just confirmed that we still need `MPN` for `pmtables` and `yspec`, but made the other suggested changes [...]

I didn't suggest that you remove this URL, just that you remove the mention of mrgval here.